### PR TITLE
wifi plugin: remove unused object, some minor style changes

### PR DESCRIPTION
**Pull request by codelegend**

### CODE REVIEW

**Comment by SilverTactic** [comment]
Hmm alright.
Btw, do I get any credit for creating the original script?

**Comment by codelegend** [comment]
original script?? I dont see anything much that actually even requires coding skill. its just an idea.
And i re-wrote that block. and the others were just punctuation changes in the messages.
You get the credit for some plugin if you make significant contribution to its design, or refactoring.

**Comment by SilverTactic** [comment]
@codelegend The script Cranham sent you to change the format of was made by me. But yeah, you're kinda right.

**Comment by codelegend** [comment]
@SilverTactic Oh sorry, i though cranham wrote that code, so thats why added his name. I suppose you could add yours too. Will add it in some future pull req.

**Comment by SilverTactic** [comment]
I see. Thanks.

File: chat-plugins/wifi.js [hunk 1]
```diff
@@ -152,10 +152,10 @@ var LotteryGiveAway = (function () {
 
 		this.reminder = '<center><div class = "broadcast-blue"><font size = 3><b>It\'s giveaway time!</b></font><br/>' +
 			'<font size = 1>Giveaway started by ' + Tools.escapeHTML(host.name) + '</font><br/><br/>' +
-			'<b>' + Tools.escapeHTML(giver.name) + '</b> will be giving away a <b>' + Tools.escapeHTML(this.prize) + '</b>!<br/>' +
+			'<b>' + Tools.escapeHTML(giver.name) + '</b> will be giving away : <b>' + Tools.escapeHTML(this.prize) + '</b>!<br/>' +
```
Comment (Zarel) [inline chat-plugins/wifi.js]: In English, it's grammatically correct to do `away: ` (no space before the `:`)

**Comment by codelegend** [comment]
fixed.

**Comment by SilverTactic** [comment]
@codelegend - Is the extra space between "Lottery Draw:" and Object.keys(this.joined).length intended in line 228?

**Pull request merged**